template_content/.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Demo current contract (+ LocalNet)",
+      "type": "python",
+      "request": "launch",
+      "module": "demo",
+      "cwd": "${fileDirname}",
```

Comment:
This means you can hit F5 when you have (say) helloworld.py open and it will run demo.py from that folder!!!! It's very slick.